### PR TITLE
add Debug to transaction builder-reported types

### PR DESCRIPTION
**Pull request by zancas**
@nuttycom here're some debug attributes

### CODE REVIEW

**Review by nuttycom — APPROVED** [review]
utACK

File: zcash_primitives/src/transaction/components/transparent/fees.rs [hunk 1]
```diff
@@ -9,7 +9,7 @@ use crate::{
 
 /// This trait provides a minimized view of a transparent input suitable for use in
 /// fee and change computation.
-pub trait InputView {
+pub trait InputView: std::fmt::Debug {
```
Comment (str4d) [inline zcash_primitives/src/transaction/components/transparent/fees.rs:12]: As a public trait, this change needs to be documented in the changelog.

File: zcash_primitives/src/transaction/components/transparent/fees.rs [hunk 2]
```diff
@@ -18,7 +18,7 @@ pub trait InputView {
 
 /// This trait provides a minimized view of a transparent output suitable for use in
 /// fee and change computation.
-pub trait OutputView {
+pub trait OutputView: std::fmt::Debug {
```
Comment (str4d) [inline zcash_primitives/src/transaction/components/transparent/fees.rs:21]: Ditto.

**Review by str4d — COMMENTED** [review]
Post-hoc utACK with necessary follow-ups.

**Pull request merged**